### ISSUE TITLE
[Reproduce] Cannot Reproduce the results.

### ISSUE THREAD

**Issue by bbhbbxwy** [littlepenguin89106/MGCAC#2]
Hello,
I have tried to reproduce the results on the FSC147 dataset using the code you provided. After downloading the CvT weights file, I directly ran the following command:
`Run python main.py --cfg=config/train.yaml`

The best validation results I obtained are: ‘MAE: 11.42 MSE: 52.01 NAE: 0.12 RSE: 2.08’
The test results are: ‘MAE: 10.42 MSE: 99.46 NAE: 0.12 RSE: 2.45’

I noticed that the MSE and RSE metrics are significantly different from the results you provided. Could you please offer some suggestions on how to improve these metrics? Specifically, I would appreciate any advice on the following aspects:

- Ensuring the correct implementation of the training and evaluation pipeline.

- Verifying the data preprocessing steps.

- Adjusting hyperparameters or training settings that might affect the results.

Thank you very much for your time and assistance. I look forward to your valuable suggestions.

**Comment by littlepenguin89106** [comment]
If you run the default setting, you should compare your result with `MGCAC` (Please refer Table 2 of our paper). Your results are similar to those presented in our paper. 
If you want to get the result of `MGCAC*`, you need to perform the test-time normalization. We provide the hyperparameter in supplementary, and you can set the values to active it. (Please refer `config/defaults.py`)
